misc/Kconfig

```diff
@@ -273,6 +276,15 @@ config BOOTLOADER_SRAM_SIZE
 	  This option specifies the amount of SRAM (measure in kB) reserved for
 	  when Zephyr is to act as a bootloader.
 
+config BOOTLOADER_MCUBOOT
+	bool
+	prompt "MCUboot bootloader support"
```

Comment:
s/MCUboot/MCUBoot/ in a few places around here too

Reply:
I'm wrong about this as discussed elsewhere; MCUboot is correct.

Reply:
N/A